dms_attachment_link/static/src/xml/chatter.xml

```diff
@@ -1,16 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<!-- Copyright 2023 Tecnativa - Víctor Martínez
+<!-- Copyright 2023-2024 Tecnativa - Víctor Martínez
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <templates id="template" xml:space="preserve">
-    <t t-inherit="mail.AttachmentBox" t-inherit-mode="extension" owl="1">
-        <xpath expr="//div[hasclass('o_AttachmentBox_content')]" position="attributes">
-            <attribute name="class" add="d-block text-center" separator=" " />
-        </xpath>
-        <xpath expr="//button[hasclass('o_AttachmentBox_buttonAdd')]" position="after">
+    <t t-inherit="mail.Chatter" t-inherit-mode="extension">
+        <xpath expr="//FileUploader//button" position="after">
             <button
                 class="o_add_dms_file_button btn btn-link"
                 type="button"
-                t-on-click.stop.prevent="attachmentBoxView._onAddDmsFile"
+                t-on-click="_onAddDmsFile"
```

Comment:
Why not using the `stop.prevent` API here. It's aimed to avoid boiler plate code in the js logic